src/variables.lisp

```diff
@@ -73,11 +73,11 @@
   "ultralisp_ro")
 
 (def-secret-env-var get-postgres-pass
-  "POSTGRES_PASS"
+  "POSTGRES_PASSWORDWORD"
```

Comment:
Here is a typo. Production will be broken :(

Interesting, why tests aren't broken in the PR?